src/libcore/lib.rs

```diff
@@ -129,6 +129,7 @@
 #![feature(reverse_bits)]
 #![feature(non_exhaustive)]
 #![feature(structural_match)]
+#![feature(avx512_target_feature)]
```

Comment:
What's the relevance of this feature (and the stdsimd update)?

Reply:
Without it we get:

```
error[E0658]: the target feature `avx512f` is currently unstable (see issue #44839)e
  --> src/libcore/../stdsimd/coresimd/x86/avx512f.rs:13:18
   |
13 | #[target_feature(enable = "avx512f")]
   |                  ^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(avx512_target_feature)] to the crate attributes to enable

error[E0658]: the target feature `avx512f` is currently unstable (see issue #44839)
  --> src/libcore/../stdsimd/coresimd/x86/avx512f.rs:30:18
   |
30 | #[target_feature(enable = "avx512f")]
   |                  ^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(avx512_target_feature)] to the crate attributes to enable

error[E0658]: the target feature `avx512f` is currently unstable (see issue #44839)
  --> src/libcore/../stdsimd/coresimd/x86/avx512f.rs:43:18
   |
43 | #[target_feature(enable = "avx512f")]
   |                  ^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(avx512_target_feature)] to the crate attributes to enable

error[E0658]: the target feature `avx512f` is currently unstable (see issue #44839)
  --> src/libcore/../stdsimd/coresimd/x86/avx512f.rs:55:18
   |
55 | #[target_feature(enable = "avx512f")]
   |                  ^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(avx512_target_feature)] to the crate attributes to enable

error[E0658]: the target feature `avx512f` is currently unstable (see issue #44839)
  --> src/libcore/../stdsimd/coresimd/x86/avx512f.rs:65:18
   |
65 | #[target_feature(enable = "avx512f")]
   |                  ^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(avx512_target_feature)] to the crate attributes to enable

error: aborting due to 5 previous errors
```